lib/Dialect/TritonGPU/Transforms/Pipeliner/SoftwarePipeliner.cpp

```diff
@@ -94,13 +94,7 @@ static bool pipelineLoop(scf::ForOp forOp, int numStages) {
 namespace {
 struct PipelinePass : public TritonGPUPipelineBase<PipelinePass> {
   PipelinePass() = default;
-  PipelinePass(int numStages, int numWarps, int numCTAs,
-               int computeCapability) {
-    this->numStages = numStages;
-    this->numWarps = numWarps;
-    this->numCTAs = numCTAs;
-    this->computeCapability = computeCapability;
-  }
+  PipelinePass(int numStages) { this->numStages = numStages; }
```

Comment:
Is it possible to use the constructor init list to initialize `numStages` ? 
```
  PipelinePass(int numStages) : numStages(numStages) {}
```

Reply:
no longer applicable after merging main